aws/logs_monitoring/README.md

```diff
@@ -96,8 +96,37 @@ If you can't install the Forwarder using the provided CloudFormation template, y
 2. Save your Datadog API key in AWS Secrets Manager, set environment variable `DD_API_KEY_SECRET_ARN` with the secret ARN on the Lambda function, and add the `secretsmanager:GetSecretValue` permission to the Lambda execution role.
 3. If you need to forward logs from S3 buckets, add the `s3:GetObject` permission to the Lambda execution role.
 4. Set the environment variable `DD_ENHANCED_METRICS` to `false` on the Forwarder. This stops the Forwarder from generating enhanced metrics itself, but it will still forward custom metrics from other lambdas.
-5. Configure [triggers](https://docs.datadoghq.com/integrations/amazon_web_services/?tab=allpermissions#send-aws-service-logs-to-datadog).
-6. Create an S3 bucket, and set environment variable `DD_S3_BUCKET_NAME` to the bucket name. Also provide `s3:GetObject`, `s3:PutObject`, and `s3:DeleteObject` permissions on this bucket to the Lambda execution role. This bucket is used to store the Lambda tags cache.
+5. Some AWS accounts are configured such that triggers will not automatically create resoucrce-based policies allowing Cloudwatch log groups to invoke the forwarder.
```

Comment:
There are other type of permissions people need, and some of those arn are different for China and GovCloud. I think an easier fix (considering not many people have to install manually) might be linking to https://github.com/DataDog/datadog-serverless-functions/blob/083957182b8475406d443d70391a6f5018102973/aws/logs_monitoring/template.yaml#L554 and simply say you need to add these permissions to your forwarder with those placeholders replaced?

Reply:
👍